lambda/src/main/scala/pricemigrationengine/services/Zuora.scala

```diff
@@ -21,15 +21,11 @@ trait Zuora {
       payload: ZuoraAmendmentOrderPayload
   ): ZIO[Any, ZuoraOrderFailure, Unit]
 
-  def applyAmendmentOrder_json_values(
-      subscription: ZuoraSubscription,
-      payload: Value
-  ): ZIO[Any, ZuoraOrderFailure, Unit]
-
-  def renewSubscription(
+  def applyOrderAsynchronously(
       subscriptionNumber: String,
-      payload: Value
-  ): ZIO[Any, ZuoraRenewalFailure, Unit]
+      payload: Value,
```

Comment:
This looks like an established pattern, but I wonder if the payload were a type instead of a Json object, which had a writer so that it could be converted to Json in the service implementation, that might be easier to reason about.  Also then the logging description could be mapped to the payload type and wouldn't need to be passed in here.